### ISSUE TITLE
Update Semantic Tagging Requirements

### ISSUE THREAD

**Issue by YigitSekerci** [bounswe/bounswe2023group4#173]
### Problem Definition

Semantic tagging must be one of the key features in our app; however, our requirements doesn't cover the semantic tagging requirements properly. So, I will update the requirements to cover the semantic tagging.

### Problem Context

* Requirements page in our wiki.

### Acceptance Criteria

* Update or create new requirements for semantic tagging.

### Suggested Solutions

* You may look what suggested in course for semantic tagging.

**Comment by YigitSekerci** [comment]
I have checked the requirements. I have found 3 relevant requirements related to semantic tagging. 

1.1.4.7.3.1 Members shall be able to filter the search result by tags to get polls according to their tags.
1.1.7.1 Domain Specific Point (Tag related)
1.2.1.1.1 System shall tag the polls depending on their content.

According to these rules semantic tagging feature can be achieved. Only problem may impact the our project that 1.2.1.1.1 requirement can be hard to implement. That's why I won't update the requirements but I'll ask the applicability of this requirement to the back-end team. If they are okay no need to change requirements according to my observations.
